processing_chain/src/main/resources/styles/lcz1.sld

```diff
@@ -0,0 +1,286 @@
+<?xml version="1.0" encoding="UTF-8"?><sld:StyledLayerDescriptor xmlns="http://www.opengis.net/sld" xmlns:sld="http://www.opengis.net/sld" xmlns:gml="http://www.opengis.net/gml" xmlns:ogc="http://www.opengis.net/ogc" version="1.0.0">
+  <sld:NamedLayer>
+    <sld:Name>rsu_lcz</sld:Name>
+    <sld:UserStyle>
+      <sld:Name>rsu_lcz</sld:Name>
+      <sld:FeatureTypeStyle>
+        <sld:Name>name</sld:Name>
+        <sld:Rule>
+          <sld:Name>LCZ 1: Compact high-rise</sld:Name>
+          <MaxScaleDenominator>500000</MaxScaleDenominator>
```

Comment:
Do we need to keep the max scale here ?

Reply:
Hum, IMO yes since it make no sense to display RSUs outside this scale range (geometries will be too small)

Reply:
Note  that SLD style is to display a map in GIS app not in a server with a big area

Reply:
+1 good catch. I will remove the max scale filtering

Reply:
done